modules/rte_mis_lottery/src/Services/RteLotteryHelper.php

```diff
@@ -218,12 +218,14 @@ public function getStudentGroupBySchool() {
    *   Type of lottery.
    * @param mixed $academic_session
    *   Academic Session.
+   * @param array $ids
+   *   Student Ids.
    */
-  public function getLotteryResult($type, $academic_session) {
+  public function getLotteryResult($type, $academic_session, $ids = []) {
```

Comment:
Function param type missing for all the parameters.

Reply:
okay